sources/il.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "url": "http://www.svivaaqm.net/",
+    "adapter": "envista_il",
+    "name": "Israel",
+    "country": "IL",
+    "city": "FIXME",
```

Comment:
The adapter isn't setting any `city` fields, any idea how to figure out the cities?